extension/manifest.json

```diff
@@ -92,5 +92,5 @@
     "/chrome/elements/shared/footer.htm"
   ],
   "minimum_chrome_version": "61",
-  "content_security_policy": "upgrade-insecure-requests; script-src 'self'; frame-ancestors https://mail.google.com 'self';"
+  "content_security_policy": "upgrade-insecure-requests; script-src 'self'; frame-ancestors https://mail.google.com 'self'; form-action 'none'; media-src 'none'; font-src 'none'; manifest-src 'none'; worker-src 'none';"
```

Comment:
The worker-src here says none but should say self. But that's the interesting part - I think we use a worker in the background page, so I would have expected this to fail.

Reply:
I see what you mean, here:

https://github.com/FlowCrypt/flowcrypt-browser/blob/a74b43931075ea5063331197870459dd3777e1f9/extension/js/background_page/background_page.ts#L21

And here:

https://github.com/FlowCrypt/flowcrypt-browser/blob/fa582dbcb14895537292e02cfa5609956b407685/extension/chrome/elements/compose.ts#L139

Correct? And yet, when I compose and send, nothing goes wrong. I'll look deeper into why that might be the case.

Good catch!